docs/source/recipes/creating-project.md

```diff
@@ -11,30 +11,34 @@ html_meta:
 For using Volto for a project (i.e. use Volto as a library), You should use Volto's project generator `@plone/generator-volto`. It's a boilerplate generator based in Yeoman that will provide you with the basic files and folder structure to bootstrap a Volto site. In addition to bootstrapping standalone Volto projects, it can also bootstrap Volto addons.
 
 1. Open a terminal and execute:
```

Comment:
For this numbered list, the syntax is a little tricky. Any content in a list must be indented. I also prefer to indent everything 4 spaces and have it aligned, including the content after a number or bullet. For example:

````
1.  Open a terminal and execute:

    ```shell
    npm install -g yo @plone/generator-volto
    yo @plone/volto
    ```
````

...will render nicely with the code-block within the context of item 1. Please apply the same to this list, and use `make docs-livehtml` to watch the changes as you edit them in real time.

Reply:
The code-block still does not indent properly. See the build preview:

https://deploy-preview-3165--volto.netlify.app/recipes/creating-project.html

Compare that code block to the code-blocks in https://deploy-preview-3165--volto.netlify.app/getting-started/install.html#install-nvm-nodejs-version-manager.

You can also look at the source of that file if you need further guidance.